<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeedbackReporter</class>
 <widget class="QWidget" name="FeedbackReporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>430</width>
    <height>557</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Error report</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>412</width>
     <height>611</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <item row="0" column="0">
       <widget class="QGraphicsView" name="graphicsView">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelTitle">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">TextLabel</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelMsg">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>To help us fix the problem, please send us this feedback report.</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEditComment">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="placeholderText">
       <string>What you did just before the problem occurred?</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTitle_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Your email address if you would like us to get in touch with you :</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxEmails">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
      <property name="title">
       <string>Files</string>
      </property>
      <widget class="QCheckBox" name="checkBoxFile1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string notr="true">CheckBox</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxFile2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string notr="true">CheckBox</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelFile1">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>30</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Crash log</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFile2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Preferences</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelFile3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>331</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>System informations</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxFile3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string notr="true">CheckBox</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelFile4">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>120</y>
         <width>331</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Log</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxFile4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>21</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string notr="true">CheckBox</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
